### ISSUE TITLE
Proposal: Tensorboard for FastNLP

### ISSUE THREAD

**Issue by FengZiYjun** [fastnlp/fastNLP#42]
Tensorboard is a good thing to monitor (or record) what happens during training, including loss, learning rate, model weights, and any evaluation metric. FastNLP is expected to provide a comprehensive view of the training process. Support for tensorboard would be a distinguishable feature.

There are a lot of open source projects about how to use Tensorboard in pytorch. However, FastNLP should minimize the rely on third-party packages. That's why I suggest developing such a thing by ourselves. 
 
reference:
https://github.com/lanpa/tensorboardX
https://github.com/yunjey/pytorch-tutorial/tree/master/tutorials/04-utils/tensorboard
https://github.com/torrvision/crayon
https://github.com/TeamHG-Memex/tensorboard_logger
